configuration/conf/config.properties

```diff
@@ -4,7 +4,7 @@
 
 ################### 被测数据库配置 #######################
 # 被测试的数据库，目前的格式为{name}{-version}{-insert mode}(注意-号)其全部参考值参见README文件
-DB_SWITCH=IoTDB-012-SESSION_BY_TABLET
```

Comment:
Because 0.13 is not stable now, so I think default value can be IoTDB-012-SESSION_BY_TABLET now

Reply:
got it